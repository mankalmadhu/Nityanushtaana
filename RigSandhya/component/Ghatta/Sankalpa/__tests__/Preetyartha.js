import React from 'react';
import ಪ್ರೀತ್ಯರ್ಥ from '../Preetyartha';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya', () => 'ವಿಷಯ');

test("ಓಮಾಪೋ", () => {
  expect(Renderer.create(
    <ಪ್ರೀತ್ಯರ್ಥ>
      ಉಪಾಸ್ಯೇ
    </ಪ್ರೀತ್ಯರ್ಥ>
  ).toJSON()).toMatchSnapshot();

});