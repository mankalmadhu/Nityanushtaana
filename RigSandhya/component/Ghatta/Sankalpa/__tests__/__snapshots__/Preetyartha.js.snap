// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಓಮಾಪೋ 1`] = `
<ವಿನಿಯೋಗ>
  ಮಮ ಉಪಾತ್ತ ಸಮಸ್ತ ದುರಿತಕ್ಷಯದ್ವಾರಾ ಶ್ರೀ ಪರಮೇಶ್ವರ ಪ್ರೀತ್ಯರ್ಥ೦ 
  ಉಪಾಸ್ಯೇ
</ವಿನಿಯೋಗ>
`;
